static/app/utils/discover/teamKeyTransactionField.tsx

```diff
@@ -87,7 +87,7 @@ export default function TeamKeyTransactionFieldWrapper({
   transactionName,
   ...props
 }: WrapperProps) {
-  const {projects} = useProjects();
+  const {projects} = useProjects({slugs: projectSlug ? [projectSlug] : []});
```

Comment:
similar to the profiling one, project should be empty if no projectId? thus no projects defined?